src/web/HTMLOperation.mjs

```diff
@@ -46,12 +46,16 @@ class HTMLOperation {
      * @returns {string}
      */
     toStubHtml(removeIcon) {
-        let html = "<li class='operation'";
+        // this.name is polluted with HTML if it originates from search-results, so before
+        // returning the HTML we purge this.name from any HTML for the data-name attribute
+        const name = this.name.replace(/(<([^>]+)>)/ig, "");
+
+        let html = `<li data-name="${name}" class="operation"`;
```

Comment:
While this works, I personally hate seeing stringified HTML in JavaScript; it's fragile and susceptible to tampering and injection. I prefer to let the browser handle element creation and manipulation through the `document.createElement()` method and subsequently manipulating the returned HTMLElement instance through its property methods.

Reply:
The then configured element can be added using `myDOMElement.appendChild(myNewElement)`.

Reply:
PS: I noticed it's done this way in `RecipeWaiter.mjs' addOperation()`